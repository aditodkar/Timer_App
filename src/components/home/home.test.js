import React from 'react';
import renderer from 'react-test-renderer';
import Home from './home';


it('Home component snapshot', () => {
    const snap = renderer.create(<Home/>).toJSON();
    expect(snap).toMatchSnapshot();
});
