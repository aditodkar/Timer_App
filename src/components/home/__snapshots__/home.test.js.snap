// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component snapshot 1`] = `
<div
  className="container"
>
  <div
    className="timeContainer"
  >
    <div>
      <span
        className="unit"
      >
        00
        :
      </span>
      <span
        className="unit"
      >
        00
        .
      </span>
      <span
        className="unit"
      >
        000
      </span>
    </div>
  </div>
  <div
    className="btnContainer"
  >
    <button
      className="btn btn-success"
      onClick={[Function]}
      type="button"
    >
      Start
    </button>
    <button
      className="btn btn-danger"
      disabled={true}
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;
